examples/crate_universe/WORKSPACE.bazel

```diff
@@ -1,4 +1,4 @@
-workspace(name = "rules_rust_examples_crate_universe")
+workspace(name = "crate_universe_examples")
```

Comment:
I've opened https://github.com/bazelbuild/rules_rust/issues/1120 based on this change.

Reply:
Could you please upload this change as a separate PR? I'm happy to approve right away.

Reply:
Opened https://github.com/bazelbuild/rules_rust/pull/1122